sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -239,10 +243,62 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
     this._context = context;
   }
 
-  private ensureUniqueReplyToForRequest() {
-    if (!this.isOpen()) {
-      this.replyTo = generate_uuid();
-    }
+  /**
+   * initialize link with unique this.replyTo address.
+   * @param options -
+   * @returns updated options bag that has adjusted `timeoutInMs` to account for init time
+   */
+  private async initWithUniqueReplyTo(
+    options: SendManagementRequestOptions = {}
+  ): Promise<SendManagementRequestOptions> {
+    const retryTimeoutInMs = options.timeoutInMs ?? Constants.defaultOperationTimeoutInMs;
+    const initOperationStartTime = Date.now();
+    return defaultCancellableLock.acquire(
+      this._initLock,
+      async () => {
+        managementClientLogger.verbose(
+          `{this._logPrefix} lock acquired for initializing replyTo address and link`
+        );
+        if (!this.isOpen()) {
+          this.replyTo = generate_uuid();
+          managementClientLogger.verbose(
+            `{this._logPrefix} new replyTo address: ${this.replyTo} generated`
+          );
+        }
+        const { abortSignal } = options ?? {};
+        const aborter = new AbortController();
+        const { signal } = new AbortController([
```

Comment:
I wonder if this bit of code (minting a new signal / aborter while inheriting from an existing one) could make for a good `core-util` helper method